dashboard/app/models/script_level.rb

```diff
@@ -442,6 +442,17 @@ def summarize(include_prev_next=true)
     summary
   end
 
+  def summarize_for_lesson_show
+    summary = summarize
+    summary[:id] = id
+    summary[:levels] = levels.map do |level|
+      {
+        id: level.id,
```

Comment:
It seems like there is information missing here

Reply:
what other information would you expect to see here -- are we supposed to be showing progress bubble color, for example?

Reply:
For example knowing if its a concept level or not determines the bubble shape. Knowing the level icon determines what icon shows.

Reply:
good catch, I would expect to find those here but it looks like those are put on the script level instead: https://github.com/code-dot-org/code-dot-org/blob/4117c7c5d18d7770fedd6126bd2f0ebe82ee5956/dashboard/app/models/script_level.rb#L384-L385

Reply:
ok, I see what you are saying now -- this isn't where the teacher facing lesson plan looks for this data. I will update this method to match.

Reply:
ok, some of this is showing up now: 
![Screen Shot 2020-10-19 at 5 06 06 PM](https://user-images.githubusercontent.com/8001765/96524578-86d70980-122d-11eb-809e-e5b7587f0e25.png)